definitions/infra-kubernetes_daemonset/golden_files.yml

```diff
@@ -0,0 +1,21 @@
+podsDesired:
```

Comment:
This file has a wrong name. It's named `golden_files` instead of `golden_metrics`.